go/chat/convsource.go

```diff
@@ -783,9 +783,27 @@ func (p *pullLocalResultCollector) String() string {
 	return fmt.Sprintf("[ %s: t: %d ]", p.Name(), p.num)
 }
 
+func (p *pullLocalResultCollector) realHits() (total int) {
+	for _, m := range p.Result() {
+		st, err := m.State()
+		if err != nil {
```

Comment:
could be `if err != nil || st != chat1. MessageUnboxedState_PLACEHOLDER`

Reply:
You know how much I like `switch` :)